clients/fides-js/src/lib/preferences.ts

```diff
@@ -20,52 +19,55 @@ import { patchUserPreferenceToFidesServer } from "../services/fides/api";
  * 3. Save preferences to the `fides_consent` cookie in the browser
  */
 export const updateConsentPreferences = ({
-  privacyNotices,
-  enabledPrivacyNoticeIds,
+  consentPreferencesToSave,
+  experienceHistoryId,
+  fidesApiUrl,
+  consentMethod,
+  userLocationString,
   debug = false,
 }: {
-  privacyNotices: PrivacyNotice[];
-  enabledPrivacyNoticeIds: Array<PrivacyNotice["id"]>;
+  consentPreferencesToSave: Array<SaveConsentPreference>;
+  experienceHistoryId: string;
+  fidesApiUrl: string;
+  consentMethod: ConsentMethod;
+  userLocationString: string;
   debug?: boolean;
 }) => {
   const cookie = getOrMakeFidesCookie();
```

Comment:
aside: this method reading from the cookie makes me mildly nervous as well. We're basically creating a shared global variable but not being careful with it's lifecycle - definitely think there's probably a bug here too.

Reply:
fixed by passing in the `cookie` obj to this method